code/modules/clothing/suits/utility.dm

```diff
@@ -123,4 +123,3 @@
 
 /obj/item/clothing/suit/radiation/New()
 	..()
-	slowdown_per_slot[slot_shoes] = 1.5
```

Comment:
Why was this removed?

Reply:
I believe when I originally did it, I started working in that file. I most likely accidently deleted that, will fix.